buildbot_nix/__init__.py

```diff
@@ -199,8 +211,19 @@ def run(self) -> Generator[Any, object, Any]:
                         skipped_builds_scheduler=f"{project_id}-nix-skipped-build",
                         name="build flake",
                         jobs=filtered_jobs,
+                        report_status=(self.number_of_jobs <= 2)
                     ),
-                ],
+                ] + [
+                    Trigger(
+                        waitForFinish=True,
+                        schedulerNames=[f"{project_id}-nix-build-combined",],
```

Comment:
That's cool. Is there a chance to link to the all failed builds in this one as well? I got the feedback that when there long lists of builds it becomes hard to see the failed ones.